app_dart/lib/src/service/scheduler.dart

```diff
@@ -213,8 +214,10 @@ class Scheduler {
   /// Filter [SchedulerConfig] to only the targets expected to run for the branch,
   /// and that do not have any dependencies.
   List<Target> getPostSubmitTargets(Commit commit, SchedulerConfig config) {
-    // Filter targets to only those run in postsubmit.
-    final List<Target> postsubmitTargets = config.targets.where((Target target) => target.postsubmit).toList();
+    // Filter targets to only those run in this postsubmit environment.
+    final List<Target> postsubmitTargets =
+        config.targets.where((Target target) => target.postsubmit && target.bringup != isProduction).toList();
```

Comment:
Would it be possible to avoid this "is this a specific environment" check in this case?

It would be better if the system has the same components/services in each of our environments. So rather than doing something specific/different in "prod", it would do the same thing with the correct instances in the particular environment. 

Rather than something like:

if env == prod: push to prod_DB, else, skip

do:

push to 'env'_DB

Reply:
Thanks, I updated it so it just filters `SchedulerConfig` when we first get it